backend/corpora/common/entities/collection.py

```diff
@@ -116,6 +116,9 @@ def list_attributes_in_time_range(
         def to_dict(db_object):
             _result = {}
             for _field in db_object._fields:
+                _value = getattr(db_object, _field)
+                if _value is None:
+                    continue
```

Comment:
Not sure where else this `to_dict` is used, but it seems like an API-specific change is leaking into ORM-level code. I would consider putting this filtering logic in the API endpoint impl method, for safety.  I think other code that might use this would be surprised if a key/value pair was entirely missing just because the value is None.

Reply:
Normally I would agree with you but our ORM and API logic is already mixed (for instance, a `to_dict` method shouldn't belong to an ORM at all) and I figured that it would make sense to minimize the amount of data manipulation by putting this additional filter here. I am not particularly opinionated on this though, so I am also happy to do it the way you suggested.

Reply:
Unless we're doing this on all entities consistently, I'd move it to the method impl for which it was added.

Reply:
I edited this accordingly.